setup.py

```diff
@@ -54,6 +55,7 @@ class install_command(install):
                         'none, cuda, cuda-clang, hip'),
         ('vec',   None, 'enable vectorization'),
         ('arch=', None, 'cpu architecture, e.g. haswell, skylake, armv8.2-a+sve, znver2 (default native).'),
+        ('neuroml', None, 'enable parsing neuroml morphologies in Arbor (regiures libxml)')
```

Comment:
small typo: **requires** libxml.

Reply:
Wouldn't call that small! Thanks for pointing it out.